afpy/routes/jobs.py

```diff
@@ -0,0 +1,79 @@
+from flask import abort
+from flask import Blueprint
+from flask import redirect
+from flask import render_template
+from flask import request
+from flask import url_for
+from peewee import DoesNotExist
+
+from afpy.forms.JobPost import JobPostForm
+from afpy.models.JobPost import JobPost
+from afpy.static import AFPY_ROOT
+from afpy.static import NEWS_PER_PAGE
+
+jobs_bp = Blueprint("jobs", __name__)
+
+
+@jobs_bp.route("/emplois/<int:post_id>")
+def jobs_render(post_id: int):
+    try:
+        job = JobPost.get_by_id(post_id)
+    except DoesNotExist:
+        abort(404)
+    return render_template("pages/job.html", body_id="emplois", job=job, name=job.title)
+
+
+@jobs_bp.route("/emplois/page/<int:current_page>")
+def jobs_page(current_page: int = 1):
+    submitted = request.args.get("submitted", False)
+    total_pages = (JobPost.select().where(JobPost.state == "published").count() // NEWS_PER_PAGE) + 1
+    jobs = (
+        JobPost.select()
+        .where(JobPost.state == "published")
+        .order_by(JobPost.dt_submitted.desc())
```

Comment:
Where's the index ensuring this order by is fast?

Reply:
fixed in [`424bfe5` (#58)](https://github.com/AFPy/site/pull/58/commits/424bfe5b6fa083dc0b9e1f195d13bebde14305ff)